### DISCUSSION TITLE
Missing proper script vetting

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#391] [Q&A]
Nice project!

Do you verify the scripts/urls before you include them in the templates?

Can we include for future projects which external scripts are executed? 

Thanks!

**Comment by tteck** [comment]
What do you mean by "Missing proper script vetting"?

**Reply by 2600box** [reply]
It is a valid point, you know, running random scripts from the internet and all that.

I suppose you could hash the scripts after "vetting" and at least know when there has been a change.

**Reply by tteck** [reply]
It's up to the user to do his/her own "vetting" or due diligence as I call it.

**Reply by ghost** [reply]
I understand that you can not go through all code, however would be great to state high over what the oneliner calls in terms of scripts on the host and on the VM:

### system commands on Proxmox (that would be of interest):
- adduser (examples)
- etc
- apt-get update

### For example the unifi script calls on lxc :

- https://raw.githubusercontent.com/tteck/Proxmox/main/setup/unifi-install.sh  -->
- https://get.glennr.nl/unifi/install/install_latest/unifi-latest.sh calls
-  https://get.glennr.nl/unifi/install/unifi-7.1.68.sh
